app/experimenter/static/css/experimenter.css

```diff
@@ -167,3 +167,7 @@ select {
   border-radius: 4px;
 	background-color: #F3F3F3;
 }
+
+label span.text-muted {
+    display: block;
```

Comment:
I don't think making this selector `display: block` like this makes semantic sense, since none of `label`, `span`, or `.text-muted` imply a block element. I think that you should introduce a new class for this behavior.

Reply:
I was just trying to make the rendered result not be different just because I fix the invalid HTML. Before it was:

```html
<label>
  <div class="text-muted">Optional</div>
...
```

We can change it to 


```html
<label>
  <span class="text-muted optional-marker">Optional</span>
...
```